Formula/fastfec.rb

```diff
@@ -24,10 +24,12 @@ def install
     args = [
       # Use brew's pcre
       "-Dvendored-pcre=false",
+      "--search-prefix",
     ]
-    if OS.linux?
-      args << "--search-prefix"
-      args << Formula["curl"].opt_prefix
+    args << if OS.linux?
+      Formula["curl"].opt_prefix
+    else
+      MacOS.sdk_path_if_needed
```

Comment:
Probably can get it to build by appending `/"usr"` to path since it is doing `-isystem <input>/"include"`.